lib/src/main/java/com/diffplug/spotless/java/ImportSorter.java

```diff
@@ -40,13 +40,23 @@ public String format(String raw) {
 		int firstImportLine = 0;
 		int lastImportLine = 0;
 		int line = 0;
+		boolean isMultiLineComment = false;
 		List<String> imports = new ArrayList<>();
 		while (scanner.hasNext()) {
 			line++;
 			String next = scanner.nextLine();
 			if (next == null) {
 				break;
 			}
+			//Since we have no AST, we only consider the most common use cases.
+			isMultiLineComment |= next.contains("/*");
+			if (isMultiLineComment && next.contains("*/")) {
+				isMultiLineComment = false;
+				if (!next.contains("/*")) {
+					continue;
+				}
```

Comment:
Sorry, just saw my mistake. I wanted to say:
In the old code `what.so.ever would` have been part of the refactored imports, **though** it was originally commented out.